website/public/css/shared.styl

```diff
@@ -60,3 +60,7 @@ ul
   .tasks-lists .row .col-md-3
     padding-right: 0
     padding-left: 0
+
+.flex-column
+  display: flex
```

Comment:
Oh, good question. What are those? If it's just old IE, we don't care, but
if it doesn't work on something current, that's a problem.

On Fri, Jul 24, 2015, 4:44 PM Blade Barringer <notifications@github.com>
wrote:

> In website/public/css/shared.styl
> <https://github.com/HabitRPG/habitrpg/pull/5664#discussion_r35468860>:
>
> > @@ -60,3 +60,7 @@ ul
> >    .tasks-lists .row .col-md-3
> >      padding-right: 0
> >      padding-left: 0
> > +
> > +.flex-column
> > +  display: flex
>
> Do we care about browsers that don't support flex?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/HabitRPG/habitrpg/pull/5664/files#r35468860>.
>